source/settings/telemetry.txt

```diff
@@ -0,0 +1,132 @@
+.. _compass-telemetry:
+
+===========================
+Configure Telemetry Options
+===========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+|compass-short| collects anonymous aggregated usage data to improve
+MongoDB products. |compass-short| collects this information by default,
+but you can disable this data collection.
+
+About this Task
+---------------
+
+|compass-short| tracks the following data:
+
+- The type of MongoDB deployment |compass-short| is connected to. For
+  example, Enterprise Edition, Community Edition, or Atlas Data Lake.
+
+- User actions performed in |compass-short|, such as creating
+  collections and indexes. |compass-short| only tracks the general
+  action performed, and does not track field values or namespaces.
```

Comment:
[suggestion] for active voice, although I can see this being a case where passive voice is appropriate

```suggestion
- The type of MongoDB deployment that you connect |compass-short| to. For
  example, Enterprise Edition, Community Edition, or Atlas Data Lake.

- Actions that you perform in |compass-short|, such as creating
  collections and indexes. |compass-short| only tracks the general
  action and does not track field values or namespaces.
```

Reply:
Thanks for these suggestions. I think using active voice works here, and this wording is an improvement!